### ISSUE TITLE
Creation of config files

### ISSUE THREAD

**Issue by deitrr** [deitrr/THOR#6]
Currently, thor needs to be re-compiled for any change made, even a change in initial conditions. Ideally, it would be better if all of the parameters that the user wants to change are put into a configuration/input file, so that there need only be one executable of thor on any system and it needs only to be compiled when source code is changed. I think this is do-able, but it may slow down certain steps.

**Comment by nabajour** [comment]
Yep, easy to do, we could do some benchmarking to check that there is not to much impact from moving from hardcoded values to variables, but shouldn't impact to much.

What format do we want? 

- simple text files that can be manually edited ? (simple to use for users and if we only want to set fixed variables)
- HDF5 files and a small python code (or UI) to edit them. That's better if we want to actually set the full state of the atmosphere. And as an extra bonus, if we use the same format as output files, we can restart a simulation from an output file.
Do we also want to configure the grid as startup options?

**Comment by deitrr** [comment]
I can see the advantage to being able to edit hdf5 files. It just makes it a bit more complicated. I wonder, is it possible to have both options? E.g., a "simple" set up (manually editing a text file) and an "advanced" set up (editing an hdf5 starting file). It might be best to start with the simple option, then once that is working we can try the hdf5 route. In any case, we can ask Joao for input next week.
